README.md

```diff
@@ -23,6 +23,7 @@
 [![GitHub commit activity](https://img.shields.io/github/commit-activity/m/honojs/hono)](https://github.com/honojs/hono/pulse)
 [![GitHub last commit](https://img.shields.io/github/last-commit/honojs/hono)](https://github.com/honojs/hono/commits/main)
 [![Discord badge](https://img.shields.io/discord/1011308539819597844?label=Discord&logo=Discord)](https://discord.gg/KMh2eNSdxV)
+[![codecov](https://codecov.io/github/honojs/hono/graph/badge.svg)](https://codecov.io/github/honojs/hono)
```

Comment:
It's nitpicking, but how about the following:

![CleanShot 2024-06-01 at 11 14 39@2x](https://github.com/honojs/hono/assets/10682/41024120-1c46-4332-b8de-05f46a2f9160)

This is better categorised.

Reply:
Makes sense.
Done in https://github.com/honojs/hono/pull/2881/commits/6e4ee3e3966a46fa0e839236db0e18f6a9ce4f10